docs/package.json

```diff
@@ -29,8 +29,9 @@
     "@jsdevtools/rehype-url-inspector": "^2.0.2",
     "@kiwicom/babel-plugin-orbit-components": "*",
     "@kiwicom/orbit-components": "*",
-    "@kiwicom/orbit-design-tokens": "*",
+    "@kiwicom/orbit-design-tokens": "5.0.0-beta.1",
     "@kiwicom/orbit-themer": "*",
+    "@kiwicom/orbit-tailwind-preset": "1.0.0-beta.0",
```

Comment:
Should we merge this with the beta versions? How are we going to handle this? Same question for design tokens package

Reply:
for docs it's ok to use beta versions :)